### PR TITLE
Add and run prettier

### LINKED ISSUE #26: Create Prettier file for common formatting

### PR DESCRIPTION
**Pull request by ChuckBTaylor**
Closes #26

### CODE REVIEW

File: src/components/molecules/NavList.tsx [hunk 1]
```diff
@@ -1,7 +1,12 @@
-import React from 'react';
-import {List, ListItemButton, ListItemText, ListSubheader} from '@mui/material';
+import React from 'react'
+import {
+  List,
+  ListItemButton,
+  ListItemText,
+  ListSubheader,
+} from '@mui/material'
```
Comment (kmdunn5) [inline src/components/molecules/NavList.tsx:7]: Is there a way we can avoid this with prettier? Or is this something you like? I don't like the new lines on the imports here, but technically doesn't matter.

File: src/components/atoms/video/Video.tsx [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
 // WIP
 export function Video({ url }: VideoProps) {
   return (
-    <CardMedia component={"iframe"} src={addQueryParamsToVideoUrl(url)} sx={{ aspectRatio: "16/9" }} />
-  );
+    <CardMedia
+      component={'iframe'}
+      src={addQueryParamsToVideoUrl(url)}
+      sx={{ aspectRatio: '16/9' }}
+    />
```
Comment (kmdunn5) [inline src/components/atoms/video/Video.tsx:15]: What's the number of props for a new line here? Sometimes it's unnecessary, but sometimes it's way better for legibility. I don't have a thought, I just don't want to think about it for only a couple of props.

File: src/components/templates/NavBar.tsx [hunk 3]
```diff
@@ -43,4 +49,4 @@ function NavBar (){
   )
 }
 
-export default NavBar
\ No newline at end of file
+export default NavBar
```
Comment (kmdunn5) [inline src/components/templates/NavBar.tsx:52]: Is this the PR to fix the `export default`s too?

File: tsconfig.json [hunk 4]
```diff
@@ -1,11 +1,7 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "lib": ["dom", "dom.iterable", "esnext"],
```
Comment (kmdunn5) [inline tsconfig.json:4]: I feel like I'd prefer new lines in a config file. No strong feeling here though.

**Review by kmdunn5 — CHANGES_REQUESTED** [review]
Just conversations, no necessary changes, but would prefer to think about these things before merging.

File: src/components/templates/NavBar.tsx (hunk 3 above)
Comment (ChuckBTaylor) [reply src/components/templates/NavBar.tsx:52]: It wasn't, no.

File: tsconfig.json (hunk 4 above)
Comment (ChuckBTaylor) [reply tsconfig.json:4]: I'm not seeing options for this.

File: src/components/atoms/video/Video.tsx (hunk 2 above)
Comment (ChuckBTaylor) [reply src/components/atoms/video/Video.tsx:15]: I'm not seeing options for this. Maybe there's a better solution than prettier that gives us more control

File: src/components/molecules/NavList.tsx (hunk 1 above)
Comment (ChuckBTaylor) [reply src/components/molecules/NavList.tsx:7]: I'm not seeing options for this.

**Review by kmdunn5 — APPROVED** [review]
Cool, I'll deal then.

**Pull request merged**